Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInRequest.java

```diff
@@ -161,6 +172,19 @@ public DropInRequest disableVenmo() {
     }
 
     /**
+     * If 3D Secure has been enabled in the control panel and an amount is specified in
+     * {@link DropInRequest#amount(String)}, Drop-In will request a 3D Secure verification for
+     * any new cards added by the user.
+     *
+     * @param requestThreeDSecure {@code true} to request a 3D Secure verification as part of Drop-In,
+     * {@code false} to not request a 3D Secure verification. Defaults to {@code false}.
+     */
+    public DropInRequest requestThreeDSecureVerification(boolean requestThreeDSecure) {
```

Comment:
`DropinRequest` is more or less a one-time use object, it doesn't really seem like someone would disable 3DS after enabling it. Would it make more sense for this method to take a `String amount` instead of a boolean?